src/test/java/software/amazon/awssdk/crt/test/ProxyTest.java

```diff
@@ -49,15 +49,13 @@
 # AWS_TEST_BASIC_AUTH_USERNAME - username to use when using basic authentication to the proxy
 # AWS_TEST_BASIC_AUTH_PASSWORD - password to use when using basic authentication to the proxy
 
-# AWS_TEST_TLS_CERT_PATH - file path to certificate used to initialize the tls context of the x509 provider connection
-# AWS_TEST_TLS_KEY_PATH - file path to the key used to initialize the tls context of the x509 provider connection
-# AWS_TEST_TLS_ROOT_CERT_PATH - file path to the root CA used to initialize the tls context of the x509 provider connection
-# AWS_TEST_X509_ENDPOINT - AWS account-specific endpoint to source x509 credentials from
-# AWS_TEST_X509_THING_NAME - associated name of the x509 thing
-# AWS_TEST_X509_ROLE_ALIAS - associated role alias ...
-
-# AWS_TEST_IOT_SIGNING_REGION - AWS region to make a websocket connection to
-# AWS_TEST_IOT_MQTT_ENDPOINT - AWS account-specific endpoint to connect to IoT core by
+# AWS_TEST_MQTT311_ROOTCA = The RootCA for connecting
```

Comment:
This appears to be the only place where we provide any kind of description of what these variables represent. Should we remove them from here or move this information to somewhere more meaningful?